src/pages/PlayerDetails/ClaimAccount/ClaimAccount.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import api from '../../../api/ApiHelper';
+import GoogleSignIn from '../../../components/GoogleSignIn/GoogleSignIn';
+import { wasAlreadyLoggedIn } from '../../../utils/GoogleUtils';
+import { getLoadingElement } from '../../../utils/LoadingUtils';
+
+interface Props {
+    playerUUID: string
+}
+
+let wasAlreadyLoggedInGoogle = wasAlreadyLoggedIn();
+
+function ClaimAccount(props: Props) {
+
+    let [verificationNumber, setVerificationNumber] = useState(0);
+    let [isLoggedIn, setIsLoggedIn] = useState(false);
+    let [isLoading, setIsLoading] = useState(wasAlreadyLoggedInGoogle);
+
+    function onAfterLogin() {
+        setIsLoading(true);
+        api.connectMinecraftAccount(props.playerUUID).then((verifyNumber) => {
+            setVerificationNumber(verifyNumber);
+            setIsLoggedIn(true);
+            setIsLoading(false);
+        });
+    }
+
+    return (
+        <div className="claim-account">
+            {isLoggedIn ?
+                <div>
+                    <h3>How do I claim my Minecraft Account?</h3>
+                    <p>Put a bid or an auction ending with the number shown below in the auction house <b>within 10 minutes</b>. After a short period we will be able to verify that this account belongs to you.</p>
+                    <h4>Your number: <b style={{ color: "lime" }}>{verificationNumber}</b></h4>
+                    <hr />
+                    <h3>Example</h3>
+                    <p>You can verify your account by these actions (examples):</p>
+                    <ul>
+                        <li>Bid <b>{verificationNumber}</b> coins on an auction</li>
```

Comment:
The verificationNumber should be left padded with 0 
![image](https://user-images.githubusercontent.com/16632490/130354622-971c97a4-e92b-4090-9dba-35889f37d4ea.png)